gsk4/src/color_stop.rs

```diff
@@ -2,8 +2,25 @@
 
 use gdk::RGBA;
 use glib::translate::*;
+use std::borrow::Cow;
 use std::fmt;
 
+#[derive(Debug, Default)]
+#[must_use = "The builder must be built to be used"]
```

Comment:
Please copy the auto generated documentation similar to the other builder types.